src/components/Breadcrumbs.js

```diff
@@ -53,7 +53,7 @@ export const Breadcrumbs = ({ path, thoughtsLimit, charLimit, className }) => {
       <TransitionGroup>
         {overflowArray.map((thoughtRanked, i) => {
           const subthoughts = !thoughtRanked.isOverflow ? ancestors(path, thoughtRanked) : null
-          return <CSSTransition key={i} timeout={200} classNames='fade'>
+          return <CSSTransition key={i} timeout={500} classNames='fade-700'>
```

Comment:
Should the timeout and CSS duration be the same amount?

Reply:
Yes, it should be the same ideally. I've updated it to `600ms`. The [throttled action-creator](https://github.com/cybersemics/em/blob/c954efc5607f85cac8e251d0ccc60d16b09cf3f8/src/action-creators/toggleToolbarAndBreadCrumbs.ts#L7) adds a delay of `100ms`. 
(Notion uses `700`)